resources/views/admin/category.blade.php

```diff
@@ -0,0 +1,42 @@
+@extends ('layouts.app')
+
+@section('title', 'Categories')
+
+@section('admin_mark', ' | Admin')
+
+@section('content')
+<style>
+</style>
+
+<div class="container">
+    <h3 class="mb-5">Categories
+    <?php $url_add = url('/admin/category/add'); ?>
+    <button class="btn btn-primary float-right" onclick="location.href='<?= $url_add; ?>'">Add new category</button>
+    </h3>
+
+    <section>
+        <table class="table table-hover">
+          <thead>
+              <tr><th>Title</th><th>Description</th><th>Action</th></tr>
+          </thead>
+
+          <tbody>
+              @foreach ($categories as $category)
+                  <?php
```

Comment:
you don't need to create a variable, creating variable consumes memory of the computer, you can assign it directly `href=""`